automated/utils/test-runner.py

```diff
@@ -907,6 +909,42 @@ def send_to_lava(self, data):
         self.logger.debug("lava-run: cmd: {}".format(cmd))
         subprocess.call(shlex.split(cmd))
 
+    def send_to_fiotest(self):
+        """
+        This method saves results as filesystem tree. This is required by
+        fiotest: https://github.com/foundriesio/fiotest/
+        """
+        # check if TEST_DIR variable is set
+        test_path = os.environ.get("TEST_DIR")
+        if not test_path:
```

Comment:
@mwasilew do you think it make sense to return here directly if test_path not set? When the path is not set(in most cases), printing the following two warning lines for every test probably is too much.

```
2021-11-23 14:54:24,216 - RUNNER.ResultParser: WARNING: TEST_DIR is not set
2021-11-23 14:54:24,216 - RUNNER.ResultParser: WARNING: NOT reporting result to fiotest
```

Reply:
Fine by me, I'll update the patch - change to debug